bands/site_sections/band_admin.py

```diff
@@ -12,10 +12,11 @@ def _required_message(self, params, fields):
                 'is' if len(missing) == 1 else 'are')
         return ''
 
-    def index(self, session, message=''):
+    def index(self, session, message='', filter='show-all'):
         return {
             'message': message,
-            'groups': session.query(Group).order_by('name').all()
+            'groups': session.query(Group).order_by('name').all(),
+            'groups_filter': filter
         }
 
     def add_band(self, session, message='', **params):
```

Comment:
Minor nitpick, but this should probably be `add_group`?

Reply:
Yea, I didn't want to get too crazy with renaming everything just for renaming it -- there was so much stuff already being changed that I was afraid I'd get lost! This should definitely be renamed though!